src/app/core/ui/search/search.component.html

```diff
@@ -1,71 +1,91 @@
-<div class="white mat-subtitle-2 remove-margin-bottom full-width">
-  <mat-form-field class="full-width searchbar">
+<div class="white mat-subtitle-2 remove-margin-bottom">
+  <button
+    *ngIf="mobile && !searchActive"
+    mat-icon-button
+    matTooltip="Search"
+    i18n-matTooltip="search toolbar icon tooltip"
+    (click)="toggleSearch()"
+  >
+    <fa-icon icon="search"></fa-icon>
+  </button>
+
+  <mat-form-field
+    *ngIf="!mobile || searchActive"
+    class="full-width searchbar"
+    [ngClass]="mobile ? 'mobile-search-wrapper' : ''"
```

Comment:
```suggestion
    [ngClass.mobile-search-wrapper]="mobile"
```

This is slightly shorter and easier to read (see https://stackoverflow.com/a/41974490/1473411)